.gitignore

```
@@ -139,4 +139,9 @@ dask-worker-space/
 # tmp directory
 tmp/
 
-**/.DS_Store
+# Mac OS
+.DS_Store
```

```suggestion
**/.DS_Store
```
(looks like the original here was better - I'll update the template)